cosmos-proxy/src/logger.js

```diff
@@ -24,33 +24,37 @@
 
 // Module dependencies
 var config = require('../conf/cosmos-proxy.json');
-var winston = require('winston');
+    winston = require('winston');
 
 // Global variables
-var logFileName = config.log.file_name;
-var logDatePattern = config.log.data_pattern;
-
-// Winston configuration
-winston.add(winston.transports.DailyRotateFile, {
-    filename: logFileName,
-    datePattern: logDatePattern
-});
-
-function info(message) {
-    winston.info(message);
-} // info
-
-function error(message) {
-    winston.error(message);
-} // error
-
-function warn(message) {
-    winston.warn(message);
-} // warn
-
-function debug(message) {
-    winston.debug(message);
-} // debug
+try {
```

Comment:
Instead of this `try - catch`, I'll do the following: check the validity of the `file_name` and `data_pattern` parameters, saving the validation result in a global variable. Then, within each method (`info`, `error`, etc) I'll check if such global variable is true or not. If true, the trace is logged.